sound/soc/codecs/hdac_hda.h

```diff
@@ -19,12 +19,22 @@ enum {
 	HDAC_LAST_DAI_ID = HDAC_HDMI_3_DAI_ID,
 };
 
+enum {
+	HDAC_HDA = 0,
```

Comment:
no need to fix to 0. Firstly you don't care at all. You just need two distinct numbers. Secondly it's guaranteed to be 0 anyway.

Reply:
Thanks @lyakh , that's indeed better.